packages/terra-card/src/Card.jsx

```diff
@@ -6,26 +6,47 @@ import styles from './Card.scss';
 import CardBody from './CardBody';
 
 const cx = classNames.bind(styles);
+const CardVariants = {
+  DEFAULT: 'default',
+  RAISED: 'raised',
+};
+
 
 const propTypes = {
   /**
    * Child Nodes
    */
   children: PropTypes.node,
+  /**
+   * Sets the card variant to change the style for different use cases. One of `Card.Opts.Variants.DEFAULT`,  `Card.Opts.Variants.RAISED`.
+   */
+  variant: PropTypes.oneOf(Object.values(CardVariants)),
+};
+
+const defaultProps = {
+  variant: 'default',
 };
 
 const Card = ({
   children,
+  variant,
   ...customProps
 }) => {
   const cardClassNames = cx([
     'card',
+    variant,
     customProps.className,
   ]);
 
   return <div {...customProps} className={cardClassNames}>{children}</div>;
 };
 
+const opts = {
+  Variants: CardVariants,
+};
+
 Card.propTypes = propTypes;
+Card.defaultProps = defaultProps;
 Card.Body = CardBody;
+Card.Opts = opts;
```

Comment:
For the record, I'm against exporting these props as an enum. Using a string is more concise and lowers the complexity of maintaining and creating these components. With prop type warning you will be warned if you've had a spelling mistake or pass in a non supported type.

Reply:
For the record, I agree with you. The first is easier
`<Card variant="raised" />`
vs
`<Card variant={Card.Opts.Variants.RAISED} />`

I also ran my wdio tests for 15 min only for them to fail because I used
`<Card variant={Card.Opts.RAISED} />` the first time on accident

Also it makes typing the wrong thing way worse. With the string it will just throw a prop type warning in dev and continue along its way. But with the object of object of object thing, mis typing like
`<Card id="raisedCard" variant={Card.Opt.Variants.RAISED}>`
throws a run time exception and crashes the page

![image](https://user-images.githubusercontent.com/8029578/39439716-2315f0c8-4c6e-11e8-9b37-658acb75e8eb.png)

or `<Card id="raisedCard" variant={Card.Opts.Variant.RAISED}>`
![image](https://user-images.githubusercontent.com/8029578/39439759-53775752-4c6e-11e8-9cf0-e1af99d4d171.png)

But, I don't really have a dog in this. Let me know what the consensus is to go with, and it sounds like there is an open issue to reconcile them all to be consistent anyways.